apis/python/src/tiledbsoma/_index_util.py

```diff
@@ -40,8 +40,8 @@ def tiledbsoma_build_index(
         tdb_concurrency = int(
```

Comment:
this all looks like dead code, or possibly wrong:
* if no context, it should use the default context
* the code to calculate thread count is dead code and can be removed

Reply:
You're right.